ui-v2/app/styles/components/app-view/layout.scss

```diff
@@ -20,6 +20,9 @@
 %app-view-header dt {
   font-weight: bold;
 }
+%app-view-header dd > a {
+  color: $black;
```

Comment:
@opihana @jnwright These are links, are they meant to be black/gray/blue when _not_ in hover?

Reply:
Which link is $app-view-header referencing?

Reply:
<img width="681" alt="Screen Shot 2020-04-22 at 10 17 01 AM" src="https://user-images.githubusercontent.com/19161242/79993177-77dfec00-8482-11ea-9907-e0fec6555a5a.png">
This one here. Below the header title, above the tabs.

Reply:
I _think_ those should be blue and underlined.

Reply:
let me point this out in Figma

Reply:
Black it is 😄